scripts/setup.sh

```diff
@@ -177,21 +177,21 @@ fi
 
 # Adjust path to support the default Chrome locations for Unix, Windows and Mac OS.
 if [ "$TRAVIS" == true ]; then
-  export CHROME_BIN="/usr/bin/chromium-browser"
```

Comment:
hmm, why this change?

Reply:
`which chromium` yields `/usr/bin/chromium` on my device, so the **CHROMIUM_BIN** environment variable wasn't working due to `chromium-browser` in path. 

![image](https://user-images.githubusercontent.com/20343475/63077131-bc5cd480-bf50-11e9-8576-43ce9952859b.png)

Reply:
Can you file an issue for this then? I would rather not touch this in this PR.

Reply:
Sure!